### ISSUE TITLE
Issue with k8s.io/releases/release/

### ISSUE THREAD

**Issue by dockerhk1203** [kubernetes/website#40927]
Release cycle diagram is not aligned with the release and at the bottom of the diagram release version should be upcoming next but still old is written.
For example:
if it starts with 1.24 then it should be: 1.24, .125, 1.26 and then last should be 1.27, but in the document 1.26 is written.

Release plan should be described in more details because current description doesn't talk much about life cycle. At one place it says 14 months patch support but in diagram only 8 months are visible.

**Comment by Ritikaa96** [comment]
Hi @dockerhk1203 , K8s releases version  on quarterly basis , you can check milestones and tags for specific issues for same in repo as well.
Regarding improvement, you can propose your plan and sig-release and sig-docs members can check them to accept or not.
Any kind of contribution to improve the docs or proposal is welcome.

**Comment by Ritikaa96** [comment]
Have a look at https://kubernetes.io/docs/contribute/ 
for contribution to kubernetes/website

**Comment by sftim** [comment]
I don't think that that diagram needs to be current. It's the same _process_ whatever the latest minor release happens to be.

However, the page in question is autogenerated from https://github.com/kubernetes/community/blob/master/contributors/devel/sig-release/release.md

Let's move this issue to that repo.

/transfer community
